nncf/experimental/torch/fx/nncf_graph_builder.py

```diff
@@ -121,19 +120,22 @@ def get_edge_params(
             edge tensor shape.
         """
         output_port_id = 0
```

Comment:
What differ in using output_idx and output_port_id, is it not the same in all cases?